### ISSUE TITLE
Work out what to do when ispyb is down

### ISSUE THREAD

**Issue by DominicOram** [DiamondLightSource/hyperion#896]
If ispyb goes down we should at least:
* Alert that we're waiting on it
* Queue up any writes to it
* Maybe pause before next collection?

**Comment by dperl-dls** [comment]
Queueing up writes seems like something we should do together with running the ISPyB callback in a separate process, probably a separate ticket to alerting that it's down?
